TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/base/managers/BaseInstanceManager.java

```diff
@@ -309,7 +309,8 @@ private void startServer(BaseServerObject server) {
                 throw new ServerStartException("Could not start process");
             }
 
-            //delay because we can only get the pid after 20 Millis.
+            final int[] retriesLeft = new int[]{10};
```

Comment:
Why are you using an array? Where is it initialized?

Reply:
what should i use instead?

Reply:
An integer variable?

Reply:
because atomicintegers are disgusting